### DISCUSSION TITLE
nonlocal total_length error

### DISCUSSION THREAD

**Discussion by freedomtrain** [searx/searx#2844] [Installation issue: custom]
I m receiving the following error message when installing searx on a linux server
This seems to be related to https://github.com/searx/searx/issues/2816 which is closed but there seems to be no documentation for the resolution of the issue ?

 File "searx/webapp.py", line 966
    nonlocal total_length
                        ^
SyntaxError: invalid syntax

**Comment by 0xhtml** [comment]
#2816 was moved to a discussion. Take a look there https://github.com/searx/searx/discussions/2818
